examples/tutorial/02_Plotting.ipynb

```diff
@@ -60,7 +60,6 @@
    "source": [
     "%%time\n",
     "df = pd.read_parquet('../data/earthquakes-projected.parq')\n",
-    "df.time = df.time.astype('datetime64[ns]')\n",
```

Comment:
Has the data file been updated to no longer require this? Or does HoloViews now handle it better?

Reply:
Difficult for me to tell what was the problem before, but with new versions of pandas this line was raising a warning. Removing it caused no problem as far as I could tell.

Reply:
Hmm. Very suspicious! @jlstevens and @philippjfr , any idea why that was previously required?

Reply:
I've ran the notebooks locally without any problem after this change. The test suite also passes (except on Windows, but that's another error). There's always a small change that I missed something...